lagunita/lms/templates/courseware/course_about_js_extra.html

```diff
@@ -0,0 +1,15 @@
+<script type="text/javascript">
+    var mobileWidth = 768;
```

Comment:
Investigate options for sticking this in a variable. Ideally, the same variable will be used by the sass.